Mods/SML/Source/SML/Private/Toolkit/KismetBytecodeDisassembler.cpp

```diff
@@ -1068,17 +1074,20 @@ TArray<TSharedPtr<FJsonValue>> FSMLKismetBytecodeDisassembler::SerializeFunction
 	int32 ScriptIndex = 0;
 
 	while (ScriptIndex < Script.Num()) {
-		const int32 StatementIndex = ScriptIndex;
 		TSharedPtr<FJsonObject> StatementObject = SerializeExpression(ScriptIndex);
-		
-		//Append statement index because several instructions can jump to statements (but not to separate expressions inside of statements!)
-		StatementObject->SetNumberField(TEXT("StatementIndex"), StatementIndex);
```

Comment:
This is why `StatementIndex` was added here: `SerializeExpression` calls itself

Reply:
Yes, I know and I want the information at every node, as I mentioned in the previous response.

Reply:
Ack